Fixes D.A.B. loadout name

NopemanMcHalt opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Just a tweak to the bad meme.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Just to avoid confusion.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: Anonymous
tweak: Renamed loadout name appropriately (ASSU -> DAB)
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

NopemanMcHalt committed:
Fixes loadout name, adds DAB outfit crate to cargo

For "recruitment" on-station.

Trilbyspaceclone commented on code/modules/cargo/packs/costumes_toys.dm:
```diff
@@ -81,6 +81,24 @@
+/datum/supply_pack/costumes_toys/neetcrate
+	name = "D.A.B. Outfit Crate"
+	desc = "When you want to recruit more \"troopers\" for useless activity. Enough for a whole squad of 6!"
+	cost = 1000
```
net cost of 300 seems a bit low maybe 1200?

NopemanMcHalt committed:
Don't do webedits, folks

I mean seriously it's quick but there's chance of fucking up something

NopemanMcHalt committed:
Partly Reverts "Fixes loadout name, adds DAB outf-

it crate to cargo"

This reverts commit 1bb8ff1c0ca56b1c1c49fc8c59d1e920b51d1872.
Removes cargo changes.

In other news, I can't please everyone and I hate society,
thank you for coming to my TED Talk.
